src/AppConfig.json.dist

```diff
@@ -1,4 +1,6 @@
 {
     "GOOGLE_API_KEY": "",
-    "STRIPE_PUBLISHABLE_KEY": ""
-}
\ No newline at end of file
+    "STRIPE_PUBLISHABLE_KEY": "",
+    "LOCALE": "",
```

Comment:
Can you just add default values here?

LOCALE = "fr"
COUNTRY_NAME = "fr"

Afterwards I am ok to merge